### PR TITLE
Indiana uni bug fix

### PR DESCRIPTION
**Pull request by dgamanenko**
OpenEdx bug fixes

### CODE REVIEW

**Comment by dgamanenko** [comment]
Alexander Zayats please review

File: lms/djangoapps/certificates/views/webview.py [hunk 1]
```diff
@@ -476,6 +477,7 @@ def render_cert_by_uuid(request, certificate_uuid):
     template_path="certificates/server-error.html",
     test_func=lambda request: request.GET.get('preview', None)
 )
+@login_required
```
Comment (z4y4ts) [inline lms/djangoapps/certificates/views/webview.py]: Why do we protect this public view while don't protect render_cert_by_uuid (line 462)?

File: lms/static/certificates/images/ico-audit.png [hunk 2]
```diff
@@ -0,0 +1 @@
+ico-honor.png
```
Comment (z4y4ts) [inline lms/static/certificates/images/ico-audit.png]: Plain text inside a image file.

File: lms/static/certificates/images/ico-audit.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+ico-honor.svg
```
Comment (z4y4ts) [inline lms/static/certificates/images/ico-audit.svg]: Plain text inside a image file

**Pull request closed without merging**